Misc/NEWS.d/next/Library/2019-08-26-02-29-14.bpo-37806.wR83qu.rst

```diff
@@ -0,0 +1,2 @@
+Fix infinite recursion with typing.get_type_hints.
```

Comment:
This news entry can be condensed onto one line, the max character width for reST is 80. 

Also, I would recommend adding a Sphinx role to link to the documentation for [typing.get_type_hints](https://docs.python.org/3/library/typing.html#typing.get_type_hints). This can still fit on one line (exactly 80 chars in total).

```suggestion
Fix infinite recursion with :func:`typing.get_type_hints`. Patch by hongweipeng.
```

Reply:
Ok,thanks.